css/src/tokens/containers.scss

```diff
@@ -0,0 +1,5 @@
+/**
+ * @sass-export-section="container"
+ */
+
```

Comment:
needs the variable? for now perhaps we just put:

```
$container-query-md:  480px !default;
```

Reply:
i tried this and it didnt like having a variable

Reply:
i added but the `@container` doesnt like the variable

Reply:
updated